### PR TITLE
WDP200303-25

### PR DESCRIPTION
**Pull request by morormo**
Add new component Promotion. 
The client wanted a new component so I added it.
Customer can change all colors and photos in settings. Content can change in dataStore

### CODE REVIEW

File: src/components/features/Promotion/Promotion.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  <div className={styles.root}>
+    <div className='container'>
+      <div className='row'>
+        <div className={`col-6 ${styles.leftBox}`}>
+          <div className={`col text-justify ${styles.boxCenter}`}>
+            <div className={`row ${styles.textBox}`}>
+              <div className='col align-self-end'>
+                <p>{leftBanner.title}</p>
+                <p>{leftBanner.subtitle}</p>
+                <p className={styles.discount}>{leftBanner.discount}</p>
```
Comment (Varenthein) [inline src/components/features/Promotion/Promotion.js]: Why do you use paragraphs here? There are clearly headings.

File: src/components/features/Promotion/Promotion.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+          </div>
+        </div>
+        <div className={'col-6'}>
+          <div className={`col-12  ${styles.rightBox}`}>
+            <div className={`row justify-content-around ${styles.textBox}`}>
+              <div className='col align-self-center text-center'>
+                <p>
+                  <b>{topBanner.bold}</b>
+                  {topBanner.title}{' '}
+                </p>
```
Comment (Varenthein) [inline src/components/features/Promotion/Promotion.js]: The same as before. Consider using headings.

Also, `<b>` tag is obsoleted. We shouldn't use tags that determine style by name. Use `strong` or `span` and style it accordingly.

File: src/components/features/Promotion/Promotion.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+                <p>{topBanner.cost}</p>
+              </div>
+            </div>
+          </div>
+          <div className={`col-12 ${styles.rightBox}`}>
+            <div className={`row justify-content-end ${styles.textBox}`}>
+              <div className='col align-self-center text-right'>
+                <p>
+                  <b>{bottomBanner.bold}</b>
+                  {bottomBanner.title}
```
Comment (Varenthein) [inline src/components/features/Promotion/Promotion.js]: The same. Dont't use `b`.

File: src/data/dataStore.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
+export const leftBanner = {
+  title: 'GUEST ROOM',
```
Comment (Varenthein) [inline src/data/dataStore.js]: Please use more specific names here. Maybe create a whole new object? For example:

```
export const promoSection = {
 leftBanner: {
   ...
},
 topBanner: {
 ...
}
...
}
```

File: src/styles/settings.scss [hunk 5]
```diff
@@ -77,3 +81,21 @@ $mb-link-active: $white;
...
+$p-leftbox-bg: $eight-light-gray;
+$p-boxcenter-bg: $second-dark-gray-opacity;
+$p-textcolor: $white;
+$p-bordercolor: $white;
+$p-rightbox-first-bg: $ninth-light-gray;
+$p-rightbox-second-bg: $tenth-light-gray;
+
+
+$image-1: '/images/sofa.JPG';
+$image-2: '/images/chair.JPG';
```
Comment (Varenthein) [inline src/styles/settings.scss]: Use more specific names for variables. `image` say only one thing – this is image. You should also cover in name, what is its role. So maybe consider something like `ps-image-one`.

Also remember not to use numbers in variables.

And one other thing ;) Your files has extension `.jpg`, not `.JPG`.

**Comment by morormo** [comment]
To do:
-fix image name
-fix image extension

Done: 
-using more specific name for variables,
-delete number from variables name
-fix image extension in variables.

**Pull request merged**